packages/component/src/providers/Theme/ThemeProvider.tsx

```diff
@@ -108,6 +109,7 @@ const ThemeProvider = ({
       sendBoxMiddleware: mergedSendBoxMiddleware,
       sendBoxToolbarMiddleware: mergedSendBoxToolbarMiddleware,
       styleOptions: mergedStyleOptions,
+      styles,
```

Comment:
I think we should cascade/merge `styles` from other cascading `<ThemeProvider>`.